packages/qdrant-loader-core/src/qdrant_loader_core/llm/providers/bedrock_utils.py

```
@@ -0,0 +1,155 @@
+                "Bedrock embedding payload must be a list of floats"
+            )
+
+        try:
+            normalized.append([
+                float(value)
+                for value in vector
+            ])
+        except (TypeError, ValueError) as exc:
+            raise InvalidRequestError(
+                f"Invalid embedding element from Bedrock: {exc}"
+            ) from exc
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)